legal-api/migrations/versions/37acd517fe98_.py

```diff
@@ -0,0 +1,47 @@
+"""empty message
+
+Revision ID: 37acd517fe98
+Revises: 8fef256bf5d7
+Create Date: 2022-08-26 13:36:49.112650
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '37acd517fe98'
+down_revision = '8fef256bf5d7'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('businesses', sa.Column('jurisdiction', sa.String(length=10), nullable=True))
+    op.add_column('businesses', sa.Column('foreign_identifier', sa.String(length=15), nullable=True))
+    op.add_column('businesses', sa.Column('foreign_legal_name', sa.String(length=1000), nullable=True))
+    op.add_column('businesses', sa.Column('foreign_legal_type', sa.String(length=10), nullable=True))
+    op.add_column('businesses', sa.Column('foreign_incorporation_date', sa.DateTime(timezone=True), nullable=True))
+    op.add_column('businesses_version', sa.Column('jurisdiction', sa.String(length=10), nullable=True))
```

Comment:
I think it would be clearer to add **foreign_** as a prefix
```suggestion
    op.add_column('businesses_version', sa.Column('foreign_jurisdiction', sa.String(length=30), nullable=True))
```